doc/source/deploy/index.rst

```diff
@@ -1,55 +1,277 @@
+   ---------------- ----------- ----------- -------- ---------------- --------
+   linux:metrics    mongo-main  42000       YES      -
+   mongodb:metrics  mongo-main  42003       YES      localhost:27017
 
-Upgrading
-=========
+For more information about the available commands of :program:`pmm-admin add`,
+run :program:`pmm-admin add --help` in your terminal.
 
-To upgrade PMM:
+.. _deploy.updating:
 
-1. :ref:`Upgrade PMM Server <upgrade-server>`
+Updating
+================================================================================
 
-#. :ref:`Upgrade PMM Client <upgrade-client>`
-   on all hosts that you are monitoring
+When changing to a new version of |product-abbrev|, you update the
+|product-abbrev| server and each |product-abbrev| client separately.
+
+The updating procedure of your |product-abbrev| server, depends on the option
+that you selected for installing it. If you have isntalled your |product-abbrev|
+server from a :program:`docker` image, follow instruction in the
+:ref:`update-server.docker` section.
+
+If you are running *PMM Server* as a :ref:`virtual appliance <run-server-ova>`
+or using :ref:`Amazon Machine Image <run-server-ami>`, you can use the update
+button in the bottom right corner of the |product-abbrev| home page (see
+:term:`PMM Home Page`).
+
+.. figure:: ../images/update-button.png
+
+   Update your server by clicking the *Update* button on the |product-abbrev|
+   landing page.
+
+.. rubric:: Updating |product-abbrev| clients
+
+When a newer version of *PMM Client* becomes available, you can update to it
+from the Percona software repositories:
+
+* For Debian or Ubuntu::
+
+   $ sudo apt-get update && sudo apt-get install pmm-client
+
+* For RedHat or CentOS::
+
+   $ yum update pmm-client
+
+If you have installed your |product-abbrev| client manually, you need to remove
+it and then download and install a newer version:
 
 Removing
-========
+================================================================================
+
+Each |product-abbrev| client and the |product-abbrev| server are removed
+separately. First, remove all monitored services by using the
+:program:`pmm-admin remove` command (see :ref:`pmm-admin-rm`). Then you can
+remove each |product-abbrev| client and the |product-abbrev| server.
+
+.. rubric:: Removing the |product-abbrev| Client
+
+The exact procedure of removing the |product-abbrev| client, depends
+on the method of installation:
+
+- Removing a package installed by using YUM:
+
+  .. code-block:: bash
+
+     $ sudo yum remove pmm-client
+  
+- Removing a package installed by using APT:
+
+  .. code-block:: bash
 
-For more information about removing PMM, see the following:
+     $ sudo apt-get remove pmm-client
+  
+- Removing a directly installed RPM package:
 
-* :ref:`remove-server`
-* :ref:`remove-client`
+  .. code-block:: bash
+
+     $ rpm -e pmm-client
+
+- Removing a directly installed DEB package:
```

Comment:
drop "directly" - see comment on RPM above

Reply:
dropping "directly" and using "manually" which is more appropriate.